_data/payments.yml

```diff
@@ -1,9 +1,13 @@
 websites:
-    - name: Amazon Payments
-      url: https://payments.amazon.com
-      twitter: AmazonPayments
-      img: amazon.png
-      tfa: No
+    - name: Amazon Pay
+      url: https://pay.amazon.com
+      img: amazonpay.png
+      tfa: Yes
+      software: Yes
+      sms: Yes
+      exceptions:
+          text: "SMS or phone call required to enable 2FA. Enabling on Amazon.com activates 2FA on other regional Amazon sites, such as UK and DE."
+      doc: https://pay.amazon.com/us/help/201212250
```

Comment:
[Secure your account with two-step verification](https://pay.amazon.com/us/help/201212250) links to [Turning on Two-Step Verification](https://www.amazon.com/gp/help/customer/display.html?nodeId=201962420) for instructions. I think the `doc` link should instead link to `https://www.amazon.com/gp/help/customer/display.html?nodeId=201962420`.

Reply:
I discussed this with our team members and we agree with your suggestion.  It will save the user an extra click, so it will be better for the user experience.  I'll resubmit the change in a moment.